### PR TITLE
feat(route): add American Chemistry Society

### LINKED ISSUE #10232: ACS期刊最新文章
### 类型

科学期刊

### 网站地址

https://pubs.acs.org/toc/jacsat/0/0

### 网站描述

Articles ASAP (as soon as publishable) are posted online and available to view immediately after technical editing, formatting for publication, and author proofing.

### 需要生成什么内容？

标题
作者
摘要
TOC图片
发表日期

### 额外描述

/jacscat可替换为acs旗下其他刊物的id，该值可在期刊链接中找到；
标题的class可以用h5.issue-item_title > a中title属性的数据；
作者的class为ul.issue-item_loa > li > span.hlFld-ContribAuthor的given name 和first name
摘要class为span.hlFld-Abstract
TOC图片class为img.lazy
发布时间class为span.pub-date-value

ACS官方确实有提供RSS，但是官方的RSS没摘要，并且搭在feedburner上，没法不翻墙访问；x-mol代收的存在#10230 的问题，因此希望能给acs和sciencedirect一样的有路由待遇

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #10232

## 完整路由地址 / Example for the proposed route(s)

```routes
/acs/journal/jacsat
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [x] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
null items in gitub-actions preview
```xml
        <item>
            <title><![CDATA[Issue Editorial Masthead]]></title>
            <description><![CDATA[null
null]]></description>
            <pubDate>Tue, 12 Jul 2022 16:00:00 GMT</pubDate>
            <guid isPermaLink="false">10.1021/jav144i027_1595734</guid>
            <link>https://pubs.acs.org/doi/10.1021/jav144i027_1595734</link>
        </item>
        <item>
            <title><![CDATA[Issue Publication Information]]></title>
            <description><![CDATA[null
null]]></description>
            <pubDate>Tue, 12 Jul 2022 16:00:00 GMT</pubDate>
            <guid isPermaLink="false">10.1021/jav144i027_1595731</guid>
            <link>https://pubs.acs.org/doi/10.1021/jav144i027_1595731</link>
        </item>
```

**Comment by lychichem** [comment]
> null items in gitub-actions preview
> ```xml
>         <item>
>             <title><![CDATA[Issue Editorial Masthead]]></title>
>             <description><![CDATA[null
> null]]></description>
>             <pubDate>Tue, 12 Jul 2022 16:00:00 GMT</pubDate>
>             <guid isPermaLink="false">10.1021/jav144i027_1595734</guid>
>             <link>https://pubs.acs.org/doi/10.1021/jav144i027_1595734</link>
>         </item>
>         <item>
>             <title><![CDATA[Issue Publication Information]]></title>
>             <description><![CDATA[null
> null]]></description>
>             <pubDate>Tue, 12 Jul 2022 16:00:00 GMT</pubDate>
>             <guid isPermaLink="false">10.1021/jav144i027_1595731</guid>
>             <link>https://pubs.acs.org/doi/10.1021/jav144i027_1595731</link>
>         </item>
> ```

这样看来的话可能需要过滤掉issue开头的没author的内容

File: lib/v2/acs/journal.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const config = require('@/config').value;
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id ?? '';
+
+    const rootUrl = 'https://pubs.acs.org';
+    const currentUrl = `${rootUrl}/toc/${id}/current?sortBy=DATE`;
```
Comment (lychichem) [inline lib/v2/acs/journal.js:11]: Should use `${rootUrl}/toc/${id}/0/0` as it is the url for ASAP.

**Review by lychichem — CHANGES_REQUESTED** [review]

File: lib/v2/acs/journal.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/acs/journal.js:11]: Thanks 😃

**Comment by lychichem** [comment]
为什么我自己尝试添加这个pr里更改的文件到我的fork，部署到vercel上不能用呢，说我没安装chromium

**Pull request merged**